scenario6/before/ProcessBuilderTest.java

```diff
@@ -59,7 +63,19 @@
 import org.camunda.bpm.model.bpmn.instance.paradigm.looping.MultiInstanceLoopCharacteristics;
 import org.camunda.bpm.model.bpmn.instance.paradigm.messaging.Message;
 import org.camunda.bpm.model.bpmn.instance.paradigm.services.Error;
-import org.camunda.bpm.model.bpmn.instance.domain.processes.Process;
+import org.camunda.bpm.model.bpmn.instance.paradigm.activities.Activity;
```

Comment:
Ist eine Trennung von Paradigm und Domain hier nicht möglich?

Reply:
das ist in before?